scraper.rb

```diff
@@ -89,14 +89,69 @@ def scrape_list(url)
 
 def scrape_person(url)
   noko = noko_for(url)
-  data = { 
-    name:noko.css('td strong').find { |n| n.text.include? 'Dip. ' }.text.sub('Dip. ','').sub(' (LICENCIA)', '').tidy,
-    image: noko.at_css('img[src*="fotos"]/@src').text, 
+  data = {
+    name:noko.css('td strong').find { |n| n.text.include? 'Dip. ' }.text.sub('Dip. ','').sub(' (LICENCIA)', '').sub(' (no rindieron protesta)', '').sub(' (pendiente protesta)', '').tidy,
```

Comment:
This line was already straining under the weight of multiple subs — adding another couple definitely pushes it over the edge.